MapboxNavigation/NavigationViewController.swift

```diff
@@ -386,16 +386,15 @@ open class NavigationViewController: UIViewController {
     
     override open func viewDidLoad() {
         super.viewDidLoad()
+        //initialize voice controller if it hasn't been overridden
+        _ = voiceController
```

Comment:
This is a rather odd way to initialize an ivar. Could we make the ivar non-lazy instead, so that it initializes along with the view controller?

Reply:
On second thought, that would be problematic because the developer might want to set a different voice controller right after initialization.

Reply:
OK, how about we add a comment saying that we’d make the ivar non-lazy and non-optional, except that we don’t want the [access token check](https://github.com/mapbox/mapbox-speech-swift/blob/b0163852eb6e3efd36effc9893e1eab10932036a/MapboxSpeech/MapboxSpeech.swift#L90) to happen before the developer has had an opportunity to swap in a different voice controller that doesn’t rely on Mapbox services.

Maybe we should defer that access token check…